api/src/pcapi/routes/serialization/venues_serialize.py

```diff
@@ -219,12 +218,20 @@ def from_orm(cls, venue: offerers_models.Venue) -> "GetVenueResponseModel":
         # does not work when passing directly an enum instance.
         venue.venueTypeCode = venue.venueTypeCode.name if venue.venueTypeCode else None  # type: ignore [attr-defined]
         venue.nonHumanizedId = venue.id
-        venue.pricingPoint = (
-            offerers_repository.find_venue_by_id(venue.current_pricing_point_id)
-            if venue.current_pricing_point_id
-            else None
-        )
-        venue.reimbursementPointId = venue.current_reimbursement_point_id
+        now = datetime.utcnow()
+        venue.pricingPoint = None
+        for pricing_link in venue.pricing_point_links:
+            if pricing_link.timespan.lower > now:
+                continue
+            if not pricing_link.timespan.upper or pricing_link.timespan.upper < now:
```

Comment:
@francois-seguin : Si la borne supérieure du lien est dans le passé, ça veut dire que le lien était actif dans le passé. Donc n'est plus actif, donc ça n'est pas le bon lien, non ?

Ça ne serait pas plutôt : 

```suggestion
            if not pricing_link.timespan.upper or pricing_link.timespan.upper > now:
```

(c'est d'ailleurs ce que tu fais quelques lignes plus bas pour le point de remboursement)